ui-src/src/app/configurations/configurations.component.ts

```diff
@@ -24,7 +26,6 @@ export class ConfigurationsComponent implements OnInit {
     private http: Http
   ) {}
 
-
   ngOnInit() {
     this.http
```

Comment:
This needs to be moved into the configuration.service.ts

Reply:
We should use this      `this.automatedUIServices.getConfig$(this.configPath, this.xmlFileName)` if we can

Reply:
Im not too sure how to use this as it takes 2 arguments and it does not go to the desired directory

Reply:
ok then lets just add a new function in the configuration service called getSchema$. that should be alot easier todo.